api/tests/core/finance/test_api.py

```diff
@@ -1061,10 +1061,11 @@ def test_generate_payment_files(mocked_gdrive_create_file):
     assert len(cashflow.logs) == 1
     assert cashflow.logs[0].statusBefore == models.CashflowStatus.PENDING
     assert cashflow.logs[0].statusAfter == models.CashflowStatus.UNDER_REVIEW
-    gdrive_file_names = {call.args[1] for call in mocked_gdrive_create_file.call_args_list}
-    assert gdrive_file_names == {
-        "reimbursement_points.csv",
-        "payment_details.csv",
+    gdrive_filenames = {call.args[1] for call in mocked_gdrive_create_file.call_args_list}
+    gdrive_filename_prefixes = {filename.rsplit("_", 2)[0] for filename in gdrive_filenames}
+    assert gdrive_filename_prefixes == {
+        "reimbursement_points",
+        "payment_details",
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏On peut tester la partie timestamp avec `freeze_gun` aussi

Reply:
À la demande de générale, je vais utiliser `freezegun` dans les tests :)